velox/functions/lib/LikeFunctionUtils.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <string>
+
+namespace facebook::velox::functions {
+namespace likeutils {
+
+// This function checks if the string 'pattern' has wildcard characters; a
+// wildcard character could either be '_' or '%'. If wildcard characters are
+// detected in the string 'pattern', the function returns true; else it returns
+// false.
+bool checkWildcardInPattern(const std::string& pattern);
+
+// This function is called if the string 'pattern' has no wildcard characters; a
```

Comment:
You can simplify this explanation to something like

"This function is called if the string 'pattern' has no wildcard characters. For this case, this function does an exact sequential match of the input string and pattern. "

Reply:
Sure, thanks!